src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
+        {
+            var c = Point3D.Centroid(points);
+
+            var A = CreateMatrix.DenseOfRowVectors(
```

Comment:
This is also called the [scatter matrix](https://math.stackexchange.com/questions/3501135/fitting-a-plane-to-points-using-svd)

Reply:
If I understood your comment correctly, it seems to me that the [scatter matrix](https://math.stackexchange.com/questions/3501135/fitting-a-plane-to-points-using-svd) means a different matrix from the matrix `A` written in the code.

To say that these 2 matrices are different, I checked their sizes(=rows count and columns count).
The [scatter matrix](https://math.stackexchange.com/questions/3501135/fitting-a-plane-to-points-using-svd) is an 3x3 matrix, 
and the matrix `A` is an nx3 matrix,
where $n$ is the count of points.

I agree with you to give variable a descriptive name.
Do you have any ideas?

Reply:
My apologies, you are right. The scatter matrix (`M` in the paper) would be `A^T * A` in your notation. Perhaps we can call this `relativePointMatrix`, since this is exactly what it is.